### ISSUE TITLE
System Monitro Button

### ISSUE THREAD

**Issue by leoheck** [udda/multiload-ng#44]
``indicator-multiload`` has and entry in menu to open ``System Monitor`` which is welcome in ``multiload-ng``.

**Comment by leoheck** [comment]
"Task Manager" is the "System Monitor", maybe a not so good name.

**Comment by udda** [comment]
Actually most system monitors nowadays are also task managers and viceversa. So both names are correct.

While I agree with you that *system monitor* term is more common and widely used, I chose to name this entry *task manager* in order to not generate confusion, as Multiload-ng is a specialized *system monitor*, and "Start system monitor" from inside Multiload-ng could be ambiguous. But this is only my opinion, I am open to reasoned changes.

I appreciated your feedback. If you want, you are free to open a new issue about the naming of the item, it will be discussed carefully!
